
  Forgot to remove include/rttests_config.h.in in the previous patch.

Signed-off-by: Sebastien Dugue <sebastien.dugue@bull.net>
Cc: Darren Hart <dvhltc@us.ibm.com>
Cc: Tim Chavez <tinytim@us.ibm.com>
---
 testcases/realtime/include/rttests_config.h.in |   94 ------------------------
 1 files changed, 0 insertions(+), 94 deletions(-)
 delete mode 100644 testcases/realtime/include/rttests_config.h.in

diff --git a/testcases/realtime/include/rttests_config.h.in b/testcases/realtime/include/rttests_config.h.in
deleted file mode 100644
index 7c55175..0000000
--- a/testcases/realtime/include/rttests_config.h.in
+++ /dev/null
@@ -1,94 +0,0 @@
-/* include/rttests_config.h.in.  Generated from configure.ac by autoheader.  */
-
-/* debugging */
-#undef DEBUG
-
-/* Define to 1 if you have the <dirent.h> header file, and it defines `DIR'.
-   */
-#undef HAVE_DIRENT_H
-
-/* Define to 1 if you have the <inttypes.h> header file. */
-#undef HAVE_INTTYPES_H
-
-/* Define to 1 if you have the <memory.h> header file. */
-#undef HAVE_MEMORY_H
-
-/* Define to 1 if you have the <ndir.h> header file, and it defines `DIR'. */
-#undef HAVE_NDIR_H
-
-/* Define to 1 if you have the `pthread_mutexattr_setprotocol' function. */
-#undef HAVE_PTHREAD_MUTEXATTR_SETPROTOCOL
-
-/* Define to 1 if you have the `pthread_mutexattr_setrobust_np' function. */
-#undef HAVE_PTHREAD_MUTEXATTR_SETROBUST_NP
-
-/* Define to 1 if you have the <stdint.h> header file. */
-#undef HAVE_STDINT_H
-
-/* Define to 1 if you have the <stdlib.h> header file. */
-#undef HAVE_STDLIB_H
-
-/* Define to 1 if you have the <strings.h> header file. */
-#undef HAVE_STRINGS_H
-
-/* Define to 1 if you have the <string.h> header file. */
-#undef HAVE_STRING_H
-
-/* Define to 1 if you have the <sys/dir.h> header file, and it defines `DIR'.
-   */
-#undef HAVE_SYS_DIR_H
-
-/* Define to 1 if you have the <sys/ndir.h> header file, and it defines `DIR'.
-   */
-#undef HAVE_SYS_NDIR_H
-
-/* Define to 1 if you have the <sys/stat.h> header file. */
-#undef HAVE_SYS_STAT_H
-
-/* Define to 1 if you have the <sys/types.h> header file. */
-#undef HAVE_SYS_TYPES_H
-
-/* Define to 1 if you have <sys/wait.h> that is POSIX.1 compatible. */
-#undef HAVE_SYS_WAIT_H
-
-/* Define to 1 if you have the <unistd.h> header file. */
-#undef HAVE_UNISTD_H
-
-/* Name of package */
-#undef PACKAGE
-
-/* Define to the address where bug reports for this package should be sent. */
-#undef PACKAGE_BUGREPORT
-
-/* Define to the full name of this package. */
-#undef PACKAGE_NAME
-
-/* Define to the full name and version of this package. */
-#undef PACKAGE_STRING
-
-/* Define to the one symbol short name of this package. */
-#undef PACKAGE_TARNAME
-
-/* Define to the version of this package. */
-#undef PACKAGE_VERSION
-
-/* Define to 1 if you have the ANSI C header files. */
-#undef STDC_HEADERS
-
-/* Define to 1 if you can safely include both <sys/time.h> and <time.h>. */
-#undef TIME_WITH_SYS_TIME
-
-/* Version number of package */
-#undef VERSION
-
-/* Define to empty if `const' does not conform to ANSI C. */
-#undef const
-
-/* Define to `__inline__' or `__inline' if that's what the C compiler
-   calls it, or to nothing if 'inline' is not supported under any name.  */
-#ifndef __cplusplus
-#undef inline
-#endif
-
-/* Define to `unsigned int' if <sys/types.h> does not define. */
-#undef size_t
-- 
1.5.4.rc2.84.gf85fd-dirty



